.eslintrc.js

```diff
@@ -228,6 +228,7 @@ module.exports = {
         './packages/desktop-client/src/components/SidebarWithData.*',
         './packages/desktop-client/src/components/Titlebar.*',
         './packages/desktop-client/src/components/UpdateNotification.*',
+        './packages/desktop-client/src/components/accounts/Header.*',
```

Comment:
We can always circle back to this file and make it darkmode compliant. My priority here is to fix lightmode regressions.